worlds/witness/options.py

```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
 
 
 class DoorGroupings(Choice):
-    """If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
-    If set to "regional", all doors in the same general region will open at once with a single key,
-    reducing the amount of door items and complexity."""
+    """
+    If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
+    If set to "regional", all doors in the same general region will open at once with a single key, reducing the amount of door items and complexity.
+    """
     display_name = "Door Groupings"
     option_off = 0
     option_regional = 1
 
 
 class ShuffleBoat(DefaultOnToggle):
-    """If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat."""
+    """
+    If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat.

```

Comment:
You used "on" elsewhere
```suggestion
    If on, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat.
```